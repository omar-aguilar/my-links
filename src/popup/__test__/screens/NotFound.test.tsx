import { render } from '@testing-library/react';
import NotFound from '../../screens/NotFound';

describe('NotFound', () => {
  it('renders without exploding', () => {
    const { container } = render(<NotFound />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
