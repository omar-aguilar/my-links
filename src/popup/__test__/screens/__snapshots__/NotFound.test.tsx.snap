// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders without exploding 1`] = `
<div>
  Not Found
</div>
`;
